user-guide/Troubleshooting/Procedures/Cloud_Connection_Issues.md

```diff
@@ -0,0 +1,33 @@
+---
+uid: Cloud_Connection_Issues
+---
+
+# Cloud Connection Issues
+This page provides solutions for issues that may arise from connectivity problems with cloud services.
+
+## Remote Access Issues
+Follow the guide below when you are encountering connection issues with dataminer.services features and your DataMiner System.
+
+### Check Status of DxMs
+A common cause for issues with [dataminer.services](xref:AboutCloudPlatform) features and your DMS is due to the connection from the [DxMs](xref:DataMinerExtensionModules) towards dataminer.services being unavailable. 
+Resolve this by taking the following steps:
+1. Verify whether the installed [DxMs](xref:DataMinerExtensionModules) are running, otherwise start them.
+1. Make sure to have the most [recent versions](xref:DCP_change_log) of the DxMs installed to avoid dealing with issues that might have been fixed. Info about upgrading DxMs can be found [here](https://community.dataminer.services/dataminer-cloud-pack/).
+
+### Check DMS Status
+Verify the status of the DMS and ensure that the system is operational, if not, start the DMS. 
```

Comment:
Maybe here we can add something about how they can validate if the WebAPI are running (don't need to mention WebAPI) by telling them to navigate to the IP defined in MaintenanceSettings.xml and see if they can access it locally (= WebAPI works locally)